tests/classification/test_hamming_distance.py

```diff
@@ -77,13 +77,22 @@ def test_hamming_distance_class(self, ddp, dist_sync_on_step, preds, target):
 
     def test_hamming_distance_fn(self, preds, target):
         self.run_functional_metric_test(
-            preds,
-            target,
+            preds=preds,
+            target=target,
```

Comment:
is this needed as it is a required argument?

Reply:
it does not really matter, because we are so inconsistent with test arguments being args or kwargs from metric to metric